### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by unclecarrot** [andycb/AdventurerClientJS#47] [General]
I just wanted to thank you for the work you put into this!

**Comment by andycb** [comment]
You're very welcome 😊

**Comment by hesiga** [comment]
Hi!
I'm new to 3d printing and just got an MP Voxel. the software isn't the best, but your Adventurer Client is amazing and has been super helpful! I just wanted to say thank you for all your work!! I know it was a long time ago, but it's new to me so I thought Id say something, let me know if your "buy me a coffee" feature still works! ;-) ... and if you ever release an iOS version lol
